_i18n/es/resources/user-guides/Offline_Backup.md

```diff
@@ -1,37 +1,36 @@
-{% include untranslated.html %}
-## Operating Systems:  Various versions of Linux and Windows 7, 8
+## Sistemas operativos: varias versiones de Linux y Windows 7, 8
  
-### Wallet Software:  Simplewallet
+### Software de monedero:  Simplewallet
  
-#### Resource for Creating Bootable Disks:  [Linux](http://www.pendrivelinux.com/),       [Windows](https://www.microsoft.com/en-us/download/windows-usb-dvd-download-tool)
+#### Recursos para crear un disco booteable: [Linux](http://www.pendrivelinux.com/),       [Windows](https://www.microsoft.com/en-us/download/windows-usb-dvd-download-tool)
  
-#### Resource for Monero Binaries:  [Monero Binaries](https://getmonero.org/downloads/)
+#### Recursos para binarios de Monero:  [Monero Binaries](https://getmonero.org/downloads/)
  
-- Take any computer you have lying around, even your normal workstation. You may find it easier to use an older computer that has no wifi or bluetooth if you're particularly paranoid
+- Toma cualquier computadora que tengas cerca, incluso tu estaci�n de trabajo. Lo tendr�s m�s f�cil si usas una computadora vieja sin Wi-Fi o Bluetooth si eres particularmente paranoico.
+
```

Comment:
A partir de esta linea, las traducciones se pierden. Además que caigo en cuenta en algo que no se como estará organizado cuando se vaya a merge la traducción. Cuando escribes (ejemplo, la linea 10 que esta en rojo, la que dice "Take any computer you have lying around,...") y haces la traducción, escribes en la linea 9 "Toma cualquier computadora que tengas cerca,..." y dejas despues un espacio en la linea 10 y sigues con otra traducción en la linea 11 en verde y es a la linea 12 en rojo (en ingles; "Create a Linux or Windows bootable disk,...) la que le sigue esa traducción de la linea 11 en verde. Entonces queda la traducción de la linea 12 en rojo por debajo de la linea 11 en verde y no la traducción 11 en verde que deberia ser la 12 en verde debajo de la 12 en rojo. No se si esto suene confuso pero asi se ve. 
Al seguir leyendo y siguiendo el orden en la traducción, se pierde nuevamente el orden de las lineas, 13 en verde es la 14 en rojo, 14 en verde espacio, 15 en verde es la 16 en rojo, y asi sucesivamente.

Reply:
Debo revisarlo con cuidado, entendí tu punto, pero no sé por qué pasó eso, apenas reviso esto, le daré una segunda leída a todo para cambiar lo de hash power y revisar mejor la gramática. Al momento de traducir me enfoco más en traducción que en gramática y por eso pasan cosas como estas jeje.